pkg/syncthing/syncthing.go

```diff
@@ -878,6 +878,10 @@ func getParent(p *process.Process) (*process.Process, error) {
 		oktetoLog.Infof("parent name is the same: %s", name)
 		return getParent(parent)
 	}
+	if !strings.Contains(pName, "okteto") {
+		oktetoLog.Infof("okteto parent not found: %s", name)
+		return getParent(parent)
```

Comment:
Maybe the name should change to `getOktetoAncestor`

Reply:
I'm changing the whole PR to make it safer deleting process